### ISSUE TITLE
using a Library 'for' causes SyntaxError

### ISSUE THREAD

**Issue by elenadimitrova** [duaraghav8/Solium#10]
Having a 
```
using LibraryContract **for** someDataType;
```
As described here http://solidity.readthedocs.io/en/latest/contracts.html?#using-for

causes a SyntaxError:
```
Error: An error occured while parsing the source code:
SyntaxError: Expected "!=", "!==", "%", "%=", "&", "&&", "&=", "*", "*=", "+", "++", "+=", ",", "-", "--", "-=", "/", "/*", "//", "/=", ";", "<", "<<", "<<=", "<=", "=", "==", "===", ">", ">=", ">>", ">>=", ">>>", ">>>=", "?", "^", "^=", "in", "instanceof", "|", "|=", "||", comment, end of line or whitespace but "f" found. Line: 25, Column: 21
An error occured while running the linter on /MyContract.sol:
```

**Comment by duaraghav8** [comment]
This occurs because solparse (the library being used internally for creating the code's AST) doesn't expect **for** at that position, the grammar is not in sync with latest Solidity version). I'll fix it and notify you once I make the new release.

Thanks for reporting this :)

**Comment by duaraghav8** [comment]
Solved in **v0.1.11**
